components/support/migration/src/main/java/mozilla/components/support/migration/AbstractMigrationService.kt

```diff
@@ -121,14 +156,4 @@ abstract class AbstractMigrationService : Service() {
 
         return NOTIFICATION_CHANNEL_ID
     }
-
-    private fun removeChannel() {
```

Comment:
I've been thinking about this recently and I think we should just keep the channel. It will not hurt much and we have a bunch of "after migration" cleanup that we want to ship eventually and removing the channel could be one of that. This will just simplify everything.